circleci/templates/maven/config_template-no-docker-v1.yml

```diff
@@ -29,11 +29,18 @@ jobs:
           command: |
             echo "export DISABLE_DOWNLOAD_PROGRESS_OPTS=-Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn" >> $BASH_ENV
             source $BASH_ENV
-      - checkout #built in GitHub repo checkout
+      - checkout # Built in GitHub repo checkout
       - run:
+          # Build code only if there is no release for this commit, otherwise update versions and prepare release
```

Comment:
This is more for my information than a correction.  Can you expound on what this means exactly?
So, code is compiled when not releasing; but when we are releasing we don't re-build but simply redo the version of the artifact?

Reply:
Yes, but after thought this should be done independent of the build step. We want a known and consistent body of work for these main build/test/scan jobs. I'm moving this maven line into the release tag job where it belongs.